cdc/sink/codec/craft.go

```diff
@@ -101,13 +101,14 @@ func (e *CraftEventBatchEncoder) Reset() {
 // SetParams reads relevant parameters for craft protocol
 func (e *CraftEventBatchEncoder) SetParams(params map[string]string) error {
 	var err error
+	maxMessageBytes, ok := params["max-message-bytes"]
+	if !ok {
+		return cerror.ErrSinkInvalidConfig.Wrap(errors.New("max-message-bytes not found"))
```

Comment:
```suggestion
		return cerror.ErrSinkInvalidConfig.GenWithStack("max-message-bytes not found")
```
This is more simple, and the error is an RFC error